modules/administration/pages/content-lifecycle.adoc

```diff
@@ -30,15 +30,56 @@ The software channels you selected should now be showing.
 . Click btn:[Attach/Detach Filters].
 . In the [guimenu]``Filters`` dialog, select the filters you want to attach to the project.
 To create a new filter, click btn:[Create new Filter].
-. Click btn:[Add new Environment].
+. Click btn:[Add Environment].
 . In the [guimenu]``Environment Lifecycle`` dialog, give the first environment a name and a description, and click btn:[Save].
-The [guimenu]``name`` field only accepts lowercase letters, numbers, periods (``.``), hyphens (``-``) and underscores (``_``).
+The [guimenu]``Name`` field only accepts lowercase letters, numbers, periods (``.``), hyphens (``-``) and underscores (``_``).
 . Continue creating environments until you have all the  environments for your lifecycle completed.
 You can select the order of the environments in the lifecycle by selecting an environment in the [guimenu]``Insert before`` field when you create it.
 
-////
+[NOTE]
+.Content Filters
+====
+Content Filters are global in your organization and can be shared in multiple projects.
+====
+
+[NOTE]
+.Adding environment in existing project
+====
+When you add an environment to an existing project that has already some sources built, the newly added environment will be populated with the content of its predecessor environment.
+====
+
 .Procedure: Using a Content Lifecycle Project
 
-. TBD ...
 
-////
+=== Building Content Project
+Once you create a project, define environments, attach sources (and possibly filters), you can built the project for the first time.
```

Comment:
`(and possibly filters)` :arrow_right: `(and possible filters)` or `(and desired filters)`?